### PR TITLE
Update DeveloperGuide.md

### PR DESCRIPTION
**Pull request by Anant1902**
Updated the DG to include more information for Mark Command.

### CODE REVIEW

**Review by belligerentbeagle — APPROVED** [review]
LGTM!

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -445,24 +445,24 @@ testers are expected to do more *exploratory* testing.
...
-1. Adding a person
+1. Adding a student
 
-   1. Prerequisites: No persons in the list.
+   1. Prerequisites: No student with NUSNet ID e0123456 or e0123457 in the contact book.
 
    1. Command 1: `addstu n/John Doe p/98765432 e/johndoe@example.com nn/e0123456 a/311, Clementi Ave 2, #02-25 
       t/friends t/owesMoney`
    1. Command 2: `addstu n/Mary Jane p/91234911 e/janemary@example.com nn/e0123457 a/312, Clementi St 1, #03-25
       t/friends t/owesTutorial2`
```
Comment (tituschewxj) [inline docs/DeveloperGuide.md:460]: Consider splitting the test cases, as currently the prerequisite is a bit ambiguous:

```
1. Adding a student with NUSNet ID e0123456
   1. Prerequisites: No student with NUSNet ID e0123456 in the contact book.
   1. Test case: `addstu n/John Doe p/98765432 e/johndoe@example.com nn/e0123456 a/311, Clementi Ave 2, #02-25 
      t/friends t/owesMoney`
1. Adding a student with NUSNet ID e0123457 
   1. Prerequisites: No student with NUSNet ID e0123457 in the contact book.
   1. Test case: `addstu n/Mary Jane p/91234911 e/janemary@example.com nn/e0123457 a/312, Clementi St 1, #03-25
      t/friends t/owesTutorial2` 
```

You could also add the expected output below the test case (like how deleting a student does it).

**Review by tituschewxj — CHANGES_REQUESTED** [review]
Thanks for updating the DG! We could try to make the test cases more clearer.

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (Anant1902) [reply docs/DeveloperGuide.md:460]: Makes sense! Will do.

**Review by tituschewxj — APPROVED** [review]
LGTM!

**Pull request merged**